<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1189</width>
    <height>596</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Gestion des employes</string>
  </property>
  <widget class="QWidget" name="employe">
   <widget class="QTabWidget" name="tabWidget">
    <property name="geometry">
     <rect>
      <x>270</x>
      <y>40</y>
      <width>731</width>
      <height>511</height>
     </rect>
    </property>
    <property name="currentIndex">
     <number>4</number>
    </property>
    <widget class="QWidget" name="AJOUTE">
     <attribute name="title">
      <string>AJOUTER EMPLOYE</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_5">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>641</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="ID_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_25">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_26">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_2">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_27">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DATE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_28">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SALAIRE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_4">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_29">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>POSTE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_ajouter">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>290</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Ajouter</string>
       </property>
      </widget>
      <widget class="QComboBox" name="poste_p">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>230</y>
         <width>121</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>chef</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>rs</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_6">
     <attribute name="title">
      <string>MODIFIER</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_6">
      <property name="geometry">
       <rect>
        <x>60</x>
        <y>40</y>
        <width>641</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="ID_3">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>40</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_30">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>80</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>NOM</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_31">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>120</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>PRENOM</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_11">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>40</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_12">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>80</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_13">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_32">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>190</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>DATE</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_33">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>150</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>SALAIRE</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_14">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>150</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_3">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>190</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_34">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>230</y>
         <width>81</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>POSTE</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_modifier">
       <property name="geometry">
        <rect>
         <x>260</x>
         <y>290</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Modifier</string>
       </property>
      </widget>
      <widget class="QComboBox" name="POSTE_p">
       <property name="geometry">
        <rect>
         <x>140</x>
         <y>230</y>
         <width>111</width>
         <height>22</height>
        </rect>
       </property>
       <item>
        <property name="text">
         <string>chef</string>
        </property>
       </item>
       <item>
        <property name="text">
         <string>RS</string>
        </property>
       </item>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab">
     <attribute name="title">
      <string>ELIMINER UN EMPLOYE</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox_7">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>30</y>
        <width>641</width>
        <height>421</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="ID_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>120</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_16">
       <property name="geometry">
        <rect>
         <x>200</x>
         <y>120</y>
         <width>113</width>
         <height>20</height>
        </rect>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_supprimer">
       <property name="geometry">
        <rect>
         <x>300</x>
         <y>190</y>
         <width>75</width>
         <height>23</height>
        </rect>
       </property>
       <property name="text">
        <string>Supprimer</string>
       </property>
      </widget>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_2">
     <attribute name="title">
      <string>CONSULTER LA LISTE</string>
     </attribute>
     <widget class="QTableView" name="tableView">
      <property name="geometry">
       <rect>
        <x>45</x>
        <y>31</y>
        <width>641</width>
        <height>341</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="tri">
      <property name="geometry">
       <rect>
        <x>250</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>tri</string>
      </property>
     </widget>
     <widget class="QPushButton" name="recherche">
      <property name="geometry">
       <rect>
        <x>400</x>
        <y>410</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>Recherche</string>
      </property>
     </widget>
     <widget class="QLineEdit" name="lineEdit_6">
      <property name="geometry">
       <rect>
        <x>390</x>
        <y>380</y>
        <width>113</width>
        <height>22</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_3">
     <attribute name="title">
      <string>CONGES</string>
     </attribute>
     <widget class="QGroupBox" name="groupBox">
      <property name="geometry">
       <rect>
        <x>10</x>
        <y>30</y>
        <width>251</width>
        <height>211</height>
       </rect>
      </property>
      <property name="title">
       <string/>
      </property>
      <widget class="QLabel" name="ID_5">
       <property name="geometry">
        <rect>
         <x>30</x>
         <y>30</y>
         <width>47</width>
         <height>13</height>
        </rect>
       </property>
       <property name="text">
        <string>ID</string>
       </property>
      </widget>
      <widget class="QLineEdit" name="lineEdit_5">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>30</y>
         <width>113</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_2">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>70</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QDateEdit" name="dateEdit_4">
       <property name="geometry">
        <rect>
         <x>90</x>
         <y>110</y>
         <width>110</width>
         <height>22</height>
        </rect>
       </property>
      </widget>
      <widget class="QLabel" name="label_2">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>70</y>
         <width>71</width>
         <height>16</height>
        </rect>
       </property>
       <property name="text">
        <string>Date debut</string>
       </property>
      </widget>
      <widget class="QLabel" name="label_3">
       <property name="geometry">
        <rect>
         <x>5</x>
         <y>110</y>
         <width>71</width>
         <height>20</height>
        </rect>
       </property>
       <property name="text">
        <string>Date retour</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_4">
       <property name="geometry">
        <rect>
         <x>150</x>
         <y>170</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>Annuler</string>
       </property>
      </widget>
      <widget class="QPushButton" name="pushButton_3">
       <property name="geometry">
        <rect>
         <x>20</x>
         <y>170</y>
         <width>93</width>
         <height>28</height>
        </rect>
       </property>
       <property name="text">
        <string>ajouter</string>
       </property>
      </widget>
     </widget>
     <widget class="QTableView" name="tableView_2">
      <property name="geometry">
       <rect>
        <x>280</x>
        <y>10</y>
        <width>421</width>
        <height>261</height>
       </rect>
      </property>
     </widget>
     <widget class="QPushButton" name="confirmer">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>250</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>confirmer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_2">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>250</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>refuser</string>
      </property>
     </widget>
     <widget class="QPushButton" name="pushButton_22">
      <property name="geometry">
       <rect>
        <x>150</x>
        <y>300</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>imprimer</string>
      </property>
     </widget>
     <widget class="QPushButton" name="supp">
      <property name="geometry">
       <rect>
        <x>30</x>
        <y>300</y>
        <width>93</width>
        <height>28</height>
       </rect>
      </property>
      <property name="text">
       <string>supprimer</string>
      </property>
     </widget>
     <widget class="QTableView" name="tableView_3">
      <property name="geometry">
       <rect>
        <x>310</x>
        <y>280</y>
        <width>331</width>
        <height>192</height>
       </rect>
      </property>
     </widget>
    </widget>
    <widget class="QWidget" name="tab_4">
     <attribute name="title">
      <string>recheche</string>
     </attribute>
    </widget>
    <widget class="QWidget" name="tab_5">
     <attribute name="title">
      <string>Page</string>
     </attribute>
    </widget>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>120</x>
      <y>40</y>
      <width>181</width>
      <height>531</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>../../../OneDrive/Bureau/Capture d’écran 2023-11-01 174534.png</pixmap>
    </property>
   </widget>
  </widget>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
